Mathlib/Algebra/Group/Pointwise/Set/Basic.lean

```diff
@@ -779,6 +779,14 @@ lemma smul_set_iUnion₂ (a : α) (s : ∀ i, κ i → Set β) :
 lemma smul_set_iInter_subset (a : α) (t : ι → Set β) : a • ⋂ i, t i ⊆ ⋂ i, a • t i :=
   image_iInter_subset _ _
 
+@[to_additive]
+lemma smul_set_sInter_subset (a : α) (S : Set (Set β)) :
+    a • ⋂₀ S ⊆ ⋂ s ∈ S, a • s := image_sInter_subset S (fun x => a • x)
+
+@[to_additive]
+lemma smul_sInter_subset (s : Set α) (T : Set (Set β)) :
+    s • ⋂₀ T ⊆ ⋂ t ∈ T, s • t := image2_sInter_subset s T (fun a x => a • x)
```

Comment:
You are missing 
```suggestion
lemma sInter_smul_subset (S : Set (Set α)) (t : Set β) :
    ⋂₀ S • t ⊆ ⋂ s ∈ S, s • t := sorry
```
and the corresponding lemmas for `mul`/`add` and ``vsub`

Reply:
Done in #17253